docs/troubleshoot/cli/troubleshoot-cli.md

```diff
@@ -19,8 +19,11 @@ These instructions apply to Zowe CLI installed on Windows, Mac OS X, and Linux s
 Collect the following information to help diagnose the issue:
 
 - Zowe CLI version installed.
+    - Issue the [Zowe CLI](../troubleshoot-check-your-zowe-version.md#zowe-cli) command to get the version number.
```

Comment:
Minor suggestion, make the link text "Zowe CLI command":

Issue the [Zowe CLI command](../troubleshoot-check-your-zowe-version.md#zowe-cli) to get the version number.